README.md

```diff
@@ -109,6 +113,31 @@ python -m tel2puml pv2puml example_above.json -jn "output_diagram"
 
 ![](docs/images/example_sequence_for_readme.svg)
 
+## Using Docker Image
+
+You can run otel2puml using the provided Docker image. This is especially useful if you want to run the tool in an isolated environment without manually managing dependencies. Here’s how to do it:
```

Comment:
provide a link to docker here (obviously they'll know but always good to provide reference for the setup etc)